src/header.rs

```diff
@@ -379,11 +1477,11 @@ impl NiftiHeader {
     pub fn affine<T>(&self) -> Matrix4<T>
     where
         T: RealField,
-        f32: SubsetOf<T>,
+        f64: SubsetOf<T>,
```

Comment:
This seems to suggest that all affine transformations will from now on use `f64` for calculations, regardless of the nature of the original data. Volume data value transformations also seem to have changed accordingly. Not sure whether this breaks any expectations to existing consumers (@nilgoyette?).